flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -520,11 +520,9 @@ private void setMeasuredDimensionForFlex(@FlexDirection int flexDirection, int w
 
     @Override
     public int getLargestMainSize() {
-        int largestSize = Integer.MIN_VALUE;
-        for (FlexLine flexLine : mFlexLines) {
-            largestSize = Math.max(largestSize, flexLine.mMainSize);
-        }
-        return largestSize;
+        return mFlexLines.stream()
+            .map(FlexLine::getMainSize)
```

Comment:
Current minSdkVersion is 14. So stream API, which requires 24 or above, can't be used.

Reply:
Ok thanks for checking.